repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java

```diff
@@ -336,6 +362,17 @@ private void mainProcess() throws IOException {
         }
         return projectInspectors;
     }
+    
+    private void inspectBuildsForBears(List<Build> buildList, String workspace) throws IOException {
+    	initWorkspace(workspace);
+    	
+    	ProjectInspector4Bears inspector;
+    	for (Build build : buildList) {
+            inspector = new ProjectInspector4Bears(build, workspace, this.serializers, null, push, steps, mode);
+            inspector.setAutoclean(clean);
+            inspector.processRepair();
```

Comment:
We should change that method name: you don't want to repair, but run the inspector.